### PR TITLE
[QS-327] Update PrivateRoute.js to match Quickstart

### PR DESCRIPTION
**Pull request by adamjmcgrath**
`PrivateRoute.js` should match https://auth0.com/docs/quickstart/spa/react#secure-the-profile-page

### CODE REVIEW

File: 01-Login/src/components/PrivateRoute.js [hunk 1]
```diff
@@ -26,9 +27,6 @@ const PrivateRoute = ({ component: Component, path, location, ...rest }) => {
 PrivateRoute.propTypes = {
   component: PropTypes.oneOfType([PropTypes.element, PropTypes.func])
     .isRequired,
-  location: PropTypes.shape({
-    pathname: PropTypes.string.isRequired
-  }).isRequired,
   path: PropTypes.oneOfType([
     PropTypes.string,
     PropTypes.arrayOf(PropTypes.string)
```
Comment (stevehobbsdev) [inline 01-Login/src/components/PrivateRoute.js]: Going the other way, I noticed that `PropTypes` is not included in the code snippet on the docs side - do you think that's worth syncing up as well?

Comment (stevehobbsdev) [reply 01-Login/src/components/PrivateRoute.js]: Same Q for sample 02 below 👇

Comment (adamjmcgrath) [reply 01-Login/src/components/PrivateRoute.js]: Yeah, happy to add it as long as you don't think it would over complicate the QS?

Comment (stevehobbsdev) [reply 01-Login/src/components/PrivateRoute.js]: Thinking about it, you're right. It adds bulk to the QS snippet without any real benefit to the auth part. Happy to leave it out.

**Comment by lbalmaceda** [comment]
@stevehobbsdev @adamjmcgrath CI passed so I'm happy to merge it if that's the blocker. Is there anything else you're waiting on?

**Comment by stevehobbsdev** [comment]
@lbalmaceda nope, feel free to merge

**Pull request merged**